pkg/buildscript/unmarshal_buildexpression.go

```diff
@@ -390,7 +389,7 @@ func transformVersion(requirements *Assignment) *FuncCall {
 					{Assignment: &Assignment{"value", o.Value}},
 				}
 			case requirementComparatorKey:
-				f.Name = cases.Title(language.English).String(strValue(o.Value))
+				f.Name = cases.Title(language.English).String(*o.Value.Str)
```

Comment:
Potential nil panic if user entered different value type.

Reply:
No, this is coming from the Platform. We are translating a platform buildexpression into a buildscript, so the user hasn't been involved yet.